shell/components/nav/WindowManager/ContainerLogs.vue

```diff
@@ -611,7 +611,7 @@ export default {
     }
 
     &.wrap-lines .msg {
-      white-space: normal;
+      white-space: pre;
```

Comment:
I think this should be
```suggestion
      white-space: pre-wrap;
```

Reply:
updated